src/components/AppListItem.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import { Button, Segment, Header} from 'semantic-ui-react'
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 const AppListItem = ({key, cohortName, cohortType, link}) => {
     return (
-       <li key={key}>
-           <p>{cohortName}</p>
-           <a href={link}>
-               {cohortType}
-           </a>
-       </li>        
+        <Segment as="li" clearing padded>
```

Comment:
just to make sure, do we need to leave key={key} for segment list elements?

Reply:
oops I forgot to include it! It's included now!